### ISSUE TITLE
Password policy is updated when CognitoUserPool trigger is added.

### ISSUE THREAD

**Issue by kizashitakata** [serverless/serverless#6574]
## Description
Sorry if I have misunderstood.
When a new trigger is added to existing user pool, the "Password strength"  is automatically updated.

ref : https://github.com/serverless/serverless/pull/6491

![スクリーンショット 2019-08-22 12 30 54](https://user-images.githubusercontent.com/29625942/63483837-9b5c2c80-c4d8-11e9-9d94-9effd27cb3d8.png)

serverless.yml is like this.
```serverless.yml
  new_func:
    handler: handler.new_func
    events:
      - cognitoUserPool:
          pool: test
          trigger: PostAuthentication
          existing: true
```

**Comment by medikoo** [comment]
@kizashitakata what version of Serverless are you using?

I believe that should have been fixed with https://github.com/serverless/serverless/pull/6519 which was published with v1.50.0

**Comment by kizashitakata** [comment]
@medikoo 
I'm sorry, the version of Serverless had been v1.49!
I updated it and confirmed the issue fixed. 
Thank you.
